src/cr-logging/Logging.cs

```diff
@@ -1,82 +1,101 @@
-﻿using System;
-using System.Configuration;
-using Microsoft.Extensions.Configuration;
-using Serilog;
-using Serilog.Core;
-using Serilog.Events;
-using Serilog.Formatting;
+﻿// <copyright file="Logging.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
 
 namespace CR.Logging
 {
+    using System;
+    using System.Configuration;
+    using Microsoft.Extensions.Configuration;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using Serilog.Formatting;
+
+    /// <summary>
+    /// Core logging.
+    /// </summary>
     public static class Logging
     {
         private static IConfigurationRoot _appConfig;
 
+        private delegate bool TryParse<in T1, T2>(T1 valueToParse, out T2 obj2);
+
+        /// <summary>
+        /// Function to setup the SeriLog Logger.
```

Comment:
How does it setup the logger (e.g. it uses the provided application config to setup the static logging class).